game/songs.cc

```diff
@@ -299,6 +299,11 @@ void Songs::reload_internal(fs::path const& parent, Cache cache) {
 			if (!regex_search(p.filename().string(), expression)) {
 				continue; //if the folder does not contain any of the requested files, ignore it
 			}
+			std::regex ignoredFiles("^(\\._.*|\\.\\#.*)$");  // skip MacOS meta, common backup files
```

Comment:
Changes look good on a first glance. I am wondering about the macos specific files like: `.DS_Store` these won't be captured by the regex, but do they really need to be captured? Since a couple of lines above we already filter out most of them?